doc/guides/examples.md

```diff
@@ -94,3 +94,38 @@ query hello {
 The query above will only work if the authenticated user has the `admin` role. You can see this rule being applied with the `@hasRole(role: "admin")` directive in [examples/keycloak/server.js](../../examples/keycloak/server.js#L22).
 
 Try change the the role to a made up role and restart the server. Try the sample query again and verify that an error is displayed.
+
+## Conflict Example
+
+`conflict/index.js` provides example for server side implementation of data conflict package that can be used with conjunction with `apollo-voyager-server` framework to detect and handle data conflicts.
```

Comment:
[Suggestion]: This could be reworded to something like:

```markdown
This example demonstrates/shows how to use the `@aerogear/vogayer-conflicts` package to detect and handle data conflicts within the resolver functions.
```